src/plugins/discourse/fetch.js

```diff
@@ -178,26 +274,32 @@ export class Fetcher implements Discourse {
     }
     failForNotOk(response);
     const json = await response.json();
-    const posts = json.post_stream.posts.map(parsePost);
+    const {posts_count: postCount} = json;
+    let posts = json.post_stream.posts.map(parsePost);
     const topic: Topic = {
       id: json.id,
+      categoryId: json.category_id,
       title: json.title,
-      timestampMs: +new Date(json.created_at),
+      timestampMs: Date.parse(json.created_at),
+      bumpedMs: json.bumped_at
```

Comment:
nit: although in practice bumped_at should never have a timestamp of 0, as a matter of habit, I would write `json.bumped_at !== null`

Reply:
Not sure what you mean exactly here. Do you mean, to fail this method when bumped_at returned by the API is null?

Reply:
Hmm, I think I wrote that nit from a place of confusion... somehow I thought `bumpMs` was a boolean and we were doing an implicit type conversion or something. Please ignore that comment. :)

Reply:
The nullable bumpedMs was really causing a lot of confusion around the codebase. In #1452 I've shifted this problem into the flow types. As some API calls have this field and others don't. I've given them different types instead of juggling with the nullable everywhere.